### PR TITLE
Clang 7 CI builds

### PR DESCRIPTION
**Pull request by offa**
Latest Clang 7 builds for Travis and Gitlab CI.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -67,3 +67,9 @@ clang-6.0:
         CONAN_CLANG_VERSIONS: "6.0"
     <<: *build-template
 
+clang-7:
+    image: lasote/conanclang7
```
Comment (bryceschober) [inline .gitlab-ci.yml]: I thought we were only going to use conanio containers. Why didn't use use conanio/clang7 as on travis?

Comment (offa) [reply .gitlab-ci.yml]: Indeed, this is wrong. Seems I didn't review my rebase :man_facepalming: 
Fix follows …

**Review by bryceschober — APPROVED** [review]

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -67,3 +67,9 @@ clang-6.0:
         CONAN_CLANG_VERSIONS: "6.0"
     <<: *build-template
 
+clang-7:
+    image: conanio/conanclang7
```
Comment (bryceschober) [inline .gitlab-ci.yml:71]: @offa I missed this in my review, and the build failed as expected: https://gitlab.com/bryceschober/conan-cpputest/-/jobs/113243435

Comment (offa) [reply .gitlab-ci.yml:71]: I should have copy & pasted this …
I'll fix this (again).

Comment (offa) [reply .gitlab-ci.yml:71]: PR is open #16

**Pull request merged**